src/shared/copy.c

```diff
@@ -794,6 +797,68 @@ static int prepare_nocow(int fdf, const char *from, int fdt, unsigned *chattr_ma
         return 0;
 }
 
+/* Copies fs-verity status.  May re-open fdt to do its job. */
+static int copy_fs_verity(int fdf, int *fdt) {
+        assert(fdf >= 0);
+        assert(fdt);
+        assert(*fdt >= 0);
+
+#ifdef FS_IOC_READ_VERITY_METADATA
+        int r = fd_verify_regular(fdf);
```

Comment:
`int r` is special to us, because it is wherever we store function return error values in it always gets declared separately at the top of functions instead of on-the-fly where first used.

Reply:
Having this outside of the `#ifdef` would have been an "unused variable" warning in the case that the definition is missing, but this is resolved by your suggestion to copy the constants.